.gitignore

```diff
@@ -7,3 +7,8 @@
 /build
 /captures
 .externalNativeBuild
+*.apk
```

Comment:
Redundant. since apk s go into /build. I dont know about rest of the extensions you added. But if you are confident. Can go ahead.